lib/kitchen/verifier/pester.rb

```diff
@@ -387,14 +520,22 @@ def restart_winrm_service
         CMD
                                      ))
       end
-
+      
       def download_test_files(state)
-        return if config[:downloads].nil?
+        if config[:downloads].nil?
+          info("Not downloading test result files from #{instance.to_str}.")
```

Comment:
I think it'd be more user-friendly to be a bit more clear that the user disabled the downloads in the configuration. Something like `Skipped downloading test result files from #{instance.to_str}; 'downloads' hash is empty.` Feel free to change up the wording till it sounds good to you.

Reply:
fixed